src/components/CenterArea/MainContents/Tweet.vue

```diff
@@ -61,6 +66,9 @@ const deleteTweet = (index) => {
       </div>
     </div>
     <pre class="tweet-text">{{ tweetContent }}</pre>
+    <div v-if="image !== ''" class="tweet-image">
```

Comment:
```suggestion
    <div v-if="!image" class="tweet-image">
```
空文字/null/undefinedとの比較の場合はその変数自体を評価するほうがシンプルに書けます。

Reply:
下記の通り修正致しました。
<div v-if="image" class="tweet-image">